css/style.css

```diff
@@ -378,18 +384,651 @@ footer {
   font-family: Poppins, sans-serif;
   font-style: normal;
   font-weight: 500;
-  font-size: 16px;
-  line-height: 24px;
+  font-size: 5vw;
   color: #92dcdc;
 }
 
 .get-resume img {
-  padding-left: 16px;
-  height: 18px;
-  width: 18px;
+  padding-left: 5vw;
+  height: 5.6vw;
+  width: 5.6vw;
   align-self: center;
 }
 
 .social img {
   align-items: center;
 }
+
+/* Desktop version */
+
+@media (min-width: 1000px) {
+  body {
+    background-image: none;
+  }
+
+  .flex-nav a {
+    display: none;
+  }
+
+  .menu {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    font-family: Poppins, sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 1.111vw;
+    color: #1a2236;
+    height: 7vw;
+    align-items: center;
+    border: 1px solid #dddfe6;
+  }
+
+  .menu li {
+    margin: 0 3.47vw;
+  }
+
+  .menu .line-hello {
+    background-color: #1a2236;
+    height: 0.14vw;
+    width: 2vw;
+  }
+
+  .menu .line-portfolio {
+    display: none;
+    background-color: #1a2236;
+    height: 0.14vw;
+    width: 2vw;
+  }
+
+  .menu .line-about {
+    display: none;
+    background-color: #1a2236;
+    height: 0.14vw;
+    width: 2vw;
+  }
+
+  .menu .line-contact {
+    display: none;
+    background-color: #1a2236;
+    height: 0.14vw;
+    width: 2vw;
+  }
+
+  main {
+    flex-direction: row;
+    height: 61vw;
+  }
+
+  .geometry img {
+    display: none;
+  }
+
+  .geometry {
+    background-image: url('../assets/img/image-geometry_3.png');
+    background-position: top right;
+    background-repeat: no-repeat;
+    background-size: 48.75vw 61vw;
+    flex: 1 1 50vw;
+  }
+
+  .main-text {
+    background-image: url('../assets/img/image_dots.png');
+    background-position: bottom right;
+    background-repeat: no-repeat;
+    background-size: 12.15vw 12.15vw;
+    height: inherit;
+    flex: 1 1 50vw;
+  }
+
+  .name h1 {
+    padding: 6.25vw 0 0 11.11vw;
+    font-family: Playfair Display, sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 7.64vw;
+    color: #1a2236;
+    line-height: 100%;
+    overflow-y: hidden;
+  }
+
+  .intro p {
+    padding: 4.16vw 6.53vw 0 11.11vw;
+    font-family: Poppins, sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 1vw;
+    letter-spacing: -0.01em;
+    color: #1a2236;
+  }
+
+  button {
+    background-color: #1a2236;
+    width: 90%;
+    padding: 1.5vw 0;
+    margin: 3.125vw 0;
+    font-family: Poppins, sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 1.25vw;
+    text-align: center;
+    color: #fff;
+  }
+
+  .btn {
```

Comment:
- [ ] Please make sure to follow the given design in all aspects such as typography, font size, font weight, colors, images, and spacings (margin and padding). Kindly fix this button's border to match the given design.
### **Your design**
![Screenshot](https://user-images.githubusercontent.com/47625407/121636394-229cda80-ca88-11eb-9eb7-02d8e375ed54.png)
### **[Original design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A9)**
![Screenshot1](https://user-images.githubusercontent.com/47625407/121636423-31838d00-ca88-11eb-9e25-01e2a26de214.png)

Reply:
fixed 👍